src/modules/token.ts

```diff
@@ -124,6 +125,32 @@ export class CurrencyModule extends ModuleWithRoles<Coin> {
     }
     await this.sendTransaction("multicall", [encoded]);
   }
+  public async getAll(): Promise<Record<string, BigNumber>> {
```

Comment:
This function needs a docstring. Just from reading it, I can say that the name doesn't communicate what it is doing.

`getAll` doesn't seem like the right name for this method. It looks like its fetching all balances and holders of a token.